src/lolex-src.js

```diff
@@ -722,6 +722,8 @@ function withGlobal(_global) {
 
         clock.reset = function reset() {
             clock.timers = {};
```

Comment:
Technically this can `delete clock.timers` and `delete clock.jobs`  to actually "reset" it since they are added lazily to the clock - though no strong felings.

Are you using `.reset` for jest?

Reply:
Then I think we can do this - it seems reasonable and since lolex is mostly used by bigger consumers rather than independently I think that a big consumer (jest) wanting the feature makes it worthwhile exposing - especially since it's already implemented and probably used but doesn't work well.

I think the current approach you're using is fine - I was just pointing it out.